src/examples/merge-with-parent.yml

```diff
@@ -0,0 +1,23 @@
+description: >
+  Merge with a defined or detected parent branch to run tests against likely 
```

Comment:
As someone not familiar with this command, I'm still not sure what this example is demonstrating exactly.